03_finetune_swift.ipynb

```diff
@@ -340,7 +340,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "use_local_mode = True # Set to true to run on local instance\n",
+    "use_local_mode = False # Set to true to run on local instance\n",
```

Comment:
Just wanted to check, this is an intentional change to the default behaviour - not an accidental check-in, right?